db/migrate/20180124092412_create_user.rb

```diff
@@ -1,11 +0,0 @@
-class CreateUser < ActiveRecord::Migration[5.1]
```

Comment:
Was this migration file supposed to be removed?

Reply:
Yep, I had used singular user in the add column migration filename and class name. I had corrected the table name within the migration and the column went to the correct table, but I think it also created another "User" table so I ended up with two tables, a "Users" and a "User"